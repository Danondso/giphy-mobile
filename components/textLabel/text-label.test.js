import React from 'react';
import TextLabel from './text-label';
import renderer from 'react-test-renderer';

test('renders Default TextLabel correctly', () => {
    const tree = renderer
      .create(<TextLabel />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('renders Dynamic TextLabel correctly', () => {
    const tree = renderer
      .create(<TextLabel label="Bonjour!"/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });