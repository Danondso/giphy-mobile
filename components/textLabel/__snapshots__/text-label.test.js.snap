// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Default TextLabel correctly 1`] = `
<Text>
  Hello!
</Text>
`;

exports[`renders Dynamic TextLabel correctly 1`] = `
<Text>
  Bonjour!
</Text>
`;
